### ISSUE TITLE
Missing alt text

### ISSUE THREAD

**Issue by sstevens2** [UW-Madison-DataScience/github-skillup-2023-09-15#4]
There is no alt text for the image in the recipe instructions.

I recommend...
A bunch of cinnamon rolls in a baking tray in an oven.

**Comment by sstevensteach** [comment]
I'll make a PR to add this to the lesson.
